### DISCUSSION TITLE
Copy from Paracon and 'Arrow up' for showing last entered line

### DISCUSSION THREAD

**Discussion by DL7NDR** [mfncooper/paracon#17] [Ideas]
Currently, Paracon allows to paste text into it, but not copy from.

If I hit "arrow up" key, the usual behaviour is to show the last entered line.

Would it be possible to add those two features?

Daniel, dl7ndr

**Comment by k6ef** [comment]
I'd like to +1 this request. I'm using Paracon to check into the Sunday Packet Net on 145.050. Sometimes it's easiest in Unproto to up arrow to resend a line because the net control didn't see it the first time. Also, it might be nice to have MACROS that can be pre-programmed for sending Unproto that is common for a sequence. i.e. If I'm Net Control it would be handy to have the most used transmissions sent with a keypress. Thanks! 73 de K6EF
